app/frontend/tests/unit/utils/permissionUtils.spec.js

```diff
@@ -0,0 +1,56 @@
+import { FormPermissions, IdentityProviders } from '@/utils/constants';
+import * as permissionUtils from '@/utils/permissionUtils';
+
+describe('checkFormSubmit', () => {
+  it('should be false when userForm is undefined', () => {
+    expect(permissionUtils.checkFormSubmit(undefined)).toBeFalsy();
+  });
+
+  it('should be false when idps is undefined', () => {
```

Comment:
you have the 'statements' mixed up with test below

Reply:
The objects I'm passing into this test are deliberately the opposite of what's passed in. For this test, I'm testing what happens when the idps attribute is undefined (the object below has { permissions: [] }, but no idps attribute). This is intended.